src/pgwire/src/protocol.rs

```diff
@@ -470,6 +483,7 @@ where
                 .await
             }
             Err(e) => {
+                self.metrics.query_status("error").inc();
```

Comment:
This would count the number of SQL errors users type. Is that what we want?

Reply:
i think it's fine - in the steady state it won't be zero, but it should be fairly low, and i think we would still get signal from if this spikes unexpectedly. i don't think we should alert on it, but it could be useful signal for tracking down issues.